linux-6.12.12.patch

```diff
@@ -733,6 +733,30 @@ index 18148e068..3da17f21e 100644
  }
  
  static int veth_convert_skb_to_xdp_buff(struct veth_rq *rq,
+diff --git a/fs/proc/generic.c b/fs/proc/generic.c
+index dbe82cf23..25ba2668d 100644
+--- a/fs/proc/generic.c
++++ b/fs/proc/generic.c
+@@ -520,6 +520,9 @@ struct proc_dir_entry *proc_mkdir(const char *name,
+ 	return proc_mkdir_data(name, 0, parent, NULL);
+ }
+ EXPORT_SYMBOL(proc_mkdir);
++#ifdef CONFIG_SECURITY_TEMPESTA
++ALLOW_ERROR_INJECTION(proc_mkdir, NULL);
++#endif
+ 
+ struct proc_dir_entry *proc_create_mount_point(const char *name)
+ {
+@@ -577,6 +580,9 @@ struct proc_dir_entry *proc_create_data(const char *name, umode_t mode,
+ 	return proc_register(parent, p);
+ }
+ EXPORT_SYMBOL(proc_create_data);
++#ifdef CONFIG_SECURITY_TEMPESTA
++ALLOW_ERROR_INJECTION(proc_create_data, NULL);
++#endif
```

Comment:
It seems there is no corresponding PR in https://github.com/tempesta-tech/linux-6.12.12-tfw